dist/gaia-header.js

```diff
@@ -1584,16 +1632,10 @@ module.exports = component.register('gaia-header', {
  * Determines whether passed element
  * contributes to the layout in gaia-header.
  *
- * Children with `[l10n-action]` get distributed
- * inside the action-button so don't occupy
- * space before the `<h1>`.
- *
  * @param  {Element}  el
  * @return {Boolean}
  */
-function contributesToLayout(el) {
-  return el.localName !== 'style' && !el.hasAttribute('l10n-action');
-}
+function contributesToLayout(el) { return el.tagName !== 'STYLE'; }
```

Comment:
This removal should not be here. No should what happened. You may need to rebase and then `npm run build` before committing again.